docs/source/tutorials/datalab/index.rst

```diff
@@ -7,4 +7,3 @@ Datalab Tutorials
    Detecting Common Data Issues with Datalab <datalab_quickstart>
    Advanced Data Auditing with Datalab <datalab_advanced>
    Text Dataset <text>
-   Tabular Data (Numeric/Categorical) <tabular>
```

Comment:
WDYT about actually keeping the tabular tutorial? 

I actually don't think this new tutorial is really a close replacement for that tabular tutorial, which is about quickly detecting issues in a tabular dataset.  Also the tabular tutorial makes it explicit that Datalab can be used for tabular data

Reply:
I am not opposed to that - and given this one isn't being added into the datalab folder, that aligns well.